models/r-models/train_german_credit.R

```diff
@@ -62,7 +62,7 @@ for (num in num_col) {
 }
 
 #model train
-model <- randomForest(x = X_train, y = as.factor(y_train), ntree = 100, type = 'classification')
+model <- randomForest(x = X_train, y = as.factor(y_train), ntree = 190, type = 'classification')
```

Comment:
See https://www.rdocumentation.org/packages/randomForest/versions/4.6-14/topics/predict.randomForest notes - should use an odd number for ntree to avoid random tie break. Well worth a comment in the code to that effect if it pans out, too.

Reply:
odd numbers seem to work. added comment